install-app.sh

```
@@ -1,23 +1,24 @@
 #!/bin/bash
 
-# Script to silently install and start the todo web app on the virtual machine. 
-# Note that all commands bellow are without sudo - that's because extention mechanism 
-# runs scripts under root user. 
+# Script to silently install and start the todo web app on the virtual machine.
+# Note that all commands bellow are without sudo - that's because extention mechanism
+# runs scripts under root user.
 
-# install system updates and isntall python3-pip package using apt. '-yq' flags are 
-# used to suppress any interactive prompts - we won't be able to confirm operation 
-# when running the script as VM extention.  
+# install system updates and isntall python3-pip package using apt. '-yq' flags are
```

The script installs `python3-pip` but does not install `git`. On many Ubuntu images `git` is not installed by default, so `git clone` will fail. Add `git` to the install command (e.g., `apt-get install -yq python3-pip git`) and consider adding `set -e` near the top to fail fast.